MKOB/kobactions.py

```diff
@@ -65,12 +65,15 @@ def doHelpAbout():
 
 # actions for control events
 
-def doOfficeID(event=None):
+def doOfficeID(event):
     kc.OfficeID = kw.varOfficeID.get()
     config.set_station(kc.OfficeID)
     config.save_config()
     km.myInternet.set_officeID(kc.OfficeID)
 
+def doCircuitCloser():
+    km.from_circuit_closer(kw.varCircuitCloser.get() == 1)
+
 def doWPM(event=None):
```

Comment:
Just curious...
Why does one have `event`, one doesn't have `event`, and one has `event=None` and none of them appear to use `event`? Is that part of the API for the GUI (I'm not too familiar with it).

Reply:
Some of the widgets pass an event parameter to their respective handlers and some don't. And some of the handlers are called directly from within the MKOB modules, and those calls never pass an event. I could have kept things simple and specified `event=None` for all the handlers, because that always works.

Reply:
Makes sense (I kind of assumed that). Might be a good change to keep the question from coming up in the future (since it's pretty common for event handlers to get an event passed in). Maybe when this file gets changed in the future.